tests/pricing_models/yieldspace_test_calc_out_given_in.py

```diff
@@ -206,12 +206,12 @@ def test_calc_out_given_in_success(self):
                     # We want to solve for the amount of bonds out given the
                     # amount of shares coming in, so we set up the problem as:
                     #
-                    #   k = (c / μ) * (μ * (z + d_z)) ** (1 - t) + (2 * y + c * z - d_y') ** (1 - t)
-                    #     = 110_000 ** (1 - T) + (300_000 - d_y') ** (1 - T)
+                    #   k = (c / mu) * (mu * (z + d_z)) ** (1 - tau) + (2 * y + c * z - d_y') ** (1 - tau)
+                    #     = 110_000 ** (t - tau + (300_000 - d_y') ** (1 - tau)
```

Comment:
Nit: This comment isn't right. It should be `(1 - tau)`. I hate to be so nitpicky, but it needs to be done right before we merge it in. If you read through your own diff, you should be able to catch problems like this before review.

Reply:
For things like this, I'd recommend reviewing the changes before you even commit as that way you won't waste your own cycles waiting for review and won't waste the reviewers cycles either.